chia/consensus/constants.py

```diff
@@ -47,3 +47,6 @@ def replace_str_to_bytes(constants: ConsensusConstants, **changes: Any) -> Conse
 
     # TODO: this is too magical here and is really only used for configuration unmarshalling
     return constants.replace(**filtered_changes)  # type: ignore[arg-type]
+
+
+__all__ = ["ConsensusConstants", "replace_str_to_bytes"]
```

Comment:
We don't do this much so I would tend to just leave the existing 'trick' with an ignore.  I need to talk to folks about getting rid of this sort of chia_rs re-exporting though in general.  I think that was a transitionary plan to reduce the size of some diffs.  If so, we can remove and deal with each case one by one to get this tidied.

Reply:
addressed